conda-store-server/conda_store_server/plugins/__init__.py

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) conda-store development team. All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+from conda_store_server._internal.plugins.lock.conda_lock import conda_lock
+
+BUILTIN_PLUGINS = [conda_lock]
```

Comment:
```suggestion
BUILTIN_PLUGINS = [
    conda_lock,
]
```
To minimize future diffs.